### PR TITLE
Add a gpu sample

### PR DESCRIPTION
**Pull request by hongye-sun**
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/575)

### CODE REVIEW

File: samples/kubeflow-tf/kubeflow-training-classification.py [hunk 1]
```diff
@@ -51,6 +51,11 @@ def kubeflow_tf_training_op(transformed_data_dir, schema: 'GcsUri[text/json]', l
         ],
         file_outputs = {'train': '/output.txt'}
     )
+    if use_gpu:
+        kubeflow_tf_training_op.image = 'gcr.io/ml-pipeline/ml-pipeline-kubeflow-tf-trainer-gpu:85c6413a2e13da4b8f198aeac1abc2f3a74fe789',
```
Comment (Ark-kun) [inline samples/kubeflow-tf/kubeflow-training-classification.py]: `.image` needs to be an string, not tuple.

**Comment by Ark-kun** [comment]
@gaoning777 I think we still need to run sample test whenever someone modifies a sample. It should probably test just that sample.

**Comment by gaoning777** [comment]
Sample tests are run in the backend no matter what. Are you inferring that we should enforce the sample test passing for merging in case samples are updated?

File: samples/kubeflow-tf/kubeflow-training-classification.py (hunk 1 above)
Comment (gaoning777) [reply samples/kubeflow-tf/kubeflow-training-classification.py]: nice catch

**Pull request merged**